<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KFileWizard</class>
 <widget class="QMainWindow" name="KFileWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>K File Wizard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>&amp;Location:</string>
        </property>
        <property name="buddy">
         <cstring>locationLine</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="locationLine">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="dirTree">
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="EntryTreeView" name="entryTree"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="fileExit"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="fileExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>About K File Wizard...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EntryTreeView</class>
   <extends>QTreeView</extends>
   <header>entryview/entrytreeview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>entryTree</tabstop>
  <tabstop>locationLine</tabstop>
  <tabstop>dirTree</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>fileExit</sender>
   <signal>triggered()</signal>
   <receiver>KFileWizard</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
